<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDataMappingComponent</class>
 <widget class="QFrame" name="CDataMappingComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="vl_DataMappingComponent">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tw_DataMappingComponent">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tb_Models">
      <attribute name="title">
       <string>Models</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_TabModels">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="BlackGui::Views::CAircraftModelView" name="tvp_Models">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_Liveries">
      <attribute name="title">
       <string>Liveries</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_TabLiveries">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="BlackGui::Views::CLiveryView" name="tvp_Liveries">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_Distributors">
      <attribute name="title">
       <string>Distributors</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_TabDistributors">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="BlackGui::Views::CDistributorView" name="tvp_Distributors">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Views::CAircraftModelView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/aircraftmodelview.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::Views::CDistributorView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/distributorview.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::Views::CLiveryView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/liveryview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
